tap_hubspot/streams.py

```diff
@@ -35,15 +35,64 @@ class MeetingsStream(HubspotStream):
     path = f"/crm/v3/objects/meetings"
     primary_keys = ["id"]
     replication_key = "updatedAt"
+    cached_schema = None
+    properties = []
 
     def get_url_params(self, context: Optional[dict], next_page_token: Optional[Any]) -> Dict[str, Any]:
         params = super().get_url_params(context, next_page_token)
-        params['properties'] = ','.join(self.get_properties_from_file())
+        params['properties'] = ','.join(self.properties)
         return params
 
+    @property
+    def schema(self) -> dict:
+        if self.cached_schema is None:
+            self.cached_schema, self.properties = self.get_custom_schema()
```

Comment:
Did you check if this was needed after all?

Reply:
Yes, for some reason it seems we do. Tbh I didn't really want to investigate, just fix